### PR TITLE
fix: [CDS-25448]: Displayed correct error message on fetching jira projects using tooltip

### PR DESCRIPTION
**Pull request by goyalsonali**
Summary:

Problem: UI fails at fetching projects in Jira and showed "Failed to fetch:400" message
Solution: Showed the exact message that is coming from BE in jira approval, create and update

 Jira Links:

https://harness.atlassian.net/browse/CDS-25448

Screenshots:

Earlier:
Screenshot 2022-02-23 at 1 11 24 PM
Screenshot 2022-02-23 at 1 11 54 PM

Now:
Screenshot 2022-02-24 at 11 52 47 AM
Screenshot 2022-02-24 at 11 53 09 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by goyalsonali** [comment]
trigger cypress

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraCreate.module.scss [hunk 1]
```diff
@@ -105,3 +105,7 @@
 .alignCenter {
   align-items: center;
 }
+.projectsFetchErrors {
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraCreate.module.scss:108]: Add some bottom margin here as from screenshots it looks like there is not enough space between error and the next field

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JiraApproval/JiraApproval.module.scss [hunk 2]
```diff
@@ -23,3 +23,8 @@
 .deploymentViewMedium {
   width: 50%;
 }
+
+.projectsFetchErrors {
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/components/PipelineSteps/Steps/JiraApproval/JiraApproval.module.scss:27]: Add some bottom margin here as from screenshots it looks like there is not enough space between error and the next field

**Review by chetanvekariya-harness — CHANGES_REQUESTED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JiraApproval/JiraApproval.module.scss (hunk 2 above)
Comment (goyalsonali) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JiraApproval/JiraApproval.module.scss:27]: Okay. I will do it

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraCreate.module.scss (hunk 1 above)
Comment (goyalsonali) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraCreate.module.scss:108]: Okay. I will do it.

**Comment by chiragp1212** [comment]
@goyalsonali can we add a tooltip and line clamp on this error? If possible, we should have the error in one line and if it overflows, use the tooltip to show the entire thing. The current screenshot looks too crowded and the BE errors can be even longer than this

**Comment by goyalsonali** [comment]
> 

Sure, I will look into it.

**Comment by vkbansal** [comment]
retrigger typecheck

**Comment by goyalsonali** [comment]
retrigger typecheck

**Comment by goyalsonali** [comment]
trigger rebase

**Comment by goyalsonali** [comment]
retrigger typecheck

**Comment by goyalsonali** [comment]
trigger rebase

**Comment by goyalsonali** [comment]
retrigger jest

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by chiragp1212 — APPROVED** [review]

**Review by vkbansal — APPROVED** [review]

**Pull request merged**